httpserver/lmq_server.cpp

```diff
@@ -220,6 +294,12 @@ void LokimqServer::init(ServiceNode* sn, RequestHandler* rh,
     lokimq_->add_category("notify", lokimq::Access{lokimq::AuthLevel::none, false, false})

```

Comment:
Can simply this `lokimq::Access{lokimq::AuthLevel::none, false, false}` to just `lokimq::AuthLevel::none`